.github/workflows/main.yml

```diff
@@ -29,8 +29,10 @@ jobs:
       - name: Install Test Dependencies
         run: |
           pip install black pytest flake8 mypy .
+          sudo apt-get install shellcheck
       - name: Lint code
         run: |
+          find . -name '*.sh' -print0 | xargs -0 shellcheck -e SC1090
```

Comment:
More info about this check https://github.com/koalaman/shellcheck/wiki/SC1090

Reply:
I skipped it mostly because it's a pain to fix. I'm not opposed to checking though.